src/APITypes.ts

```diff
@@ -1,139 +1,172 @@
 export type APIAchievement = {
+	completionInfo: null;
+	info: string;
 	name: string;
 	stars: number;
-	value: number;
 	target: number;
-	info: string;
-	completionInfo: null;
+	value: number;
 };
 
 export type APIArena = {
 	id: number;
 	name: string;
 };
 
-export type APIBadge =
-	| {
-			name: string;
-			progress: number;
-			level: number;
-			maxLevel: number;
-			target: number;
-	  }
-	| {
-			name: string;
-			progress: number;
-	  };
+export type APIBadge = APIBaseBadge | APIMultipleLevelsBadge;
+
+export type APIBaseBadge = {
+	name: string;
+	progress: number;
+};
 
 export type APIBestSeason = APISeason & {
 	id: APISeasonId;
 };
 
 export type APICard = {
-	name: string;
+	iconUrls: { medium: APIImageLink<`/cards/${number}/${string}.png`> };
 	id: number;
 	maxLevel: number;
-	iconUrls: {
-		medium: APIImageLink<`/cards/${number}/${string}.png`>;
-	};
+	name: string;
 };
 
 export type APIClan = {
-	tag: APITag;
-	name: string;
-	type: "closed" | "inviteOnly" | "open";
-	description: string;
 	badgeId: number;
+	clanChestLevel: 1;
+	clanChestMaxLevel: 0;
+	clanChestStatus: "inactive";
 	clanScore: number;
 	clanWarTrophies: number;
-	location: APILocation;
-	requiredTrophies: number;
+	description: string;
 	donationsPerWeek: number;
-	clanChestStatus: "inactive";
-	clanChestLevel: 1;
-	clanChestMaxLevel: 0;
-	members: number;
+	location: APILocation;
 	memberList: APIMember[];
+	members: number;
+	name: string;
+	requiredTrophies: number;
+	tag: APITag;
+	type: APIClanType;
 };
 
-export type APIClanPreview = {
-	tag: APITag;
-	name: string;
+export type APIClanPreview = Pick<APIClan, "badgeId" | "name" | "tag">;
+
+export type APIClanResultPreview = Pick<
+	APIClan,
+	| "badgeId"
+	| "clanScore"
+	| "clanWarTrophies"
+	| "donationsPerWeek"
+	| "location"
+	| "members"
+	| "name"
+	| "requiredTrophies"
+	| "tag"
+	| "type"
+>;
+
+export type APIClanSearchResults = APIList<APIClanResultPreview>;
+
+export type APIClanStanding = {
 	badgeId: number;
+	clanScore: number;
+	fame: number;
+	finishTime: APIDate;
+	name: string;
+	participants: APIRiverRaceParticipant[];
+	periodPoints: 0;
```

Comment:
This _should_ be the number of points earned in the current day.
```suggestion
	periodPoints: number;
```